test/e2e/metrics_util.go

```diff
@@ -131,7 +137,10 @@ func readLatencyMetrics(c *client.Client) (APIResponsiveness, error) {
 
 // Prints summary metrics for request types with latency above threshold
 // and returns number of such request types.
-func HighLatencyRequests(c *client.Client, threshold time.Duration) (int, error) {
+// For the threshold we implicitly assume:
```

Comment:
also, I'm not sure I would list specific numbers here, since you'll need to update it every time the constant changes.